com.stansassets.google-doc-connector-pro/Runtime/Localization/LocalizationClient.cs

```diff
@@ -348,6 +373,14 @@ public string GetLocalizedString(ILocalizationToken localizationToken, string la
 
             return finalText;
         }
+        
+        /// <summary>
+        /// Notify that the language settings of the spreadsheet have been changed
+        /// </summary>
+        void NotifyLanguageChanged()
+        {
+            OnLanguageChanged?.Invoke();
```

Comment:
Redundant null-check here. If you initialize `event Action` with an empty delegate (just like on Line 12), there is no need to check it for a null before invocation.